jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/completion/HandlerCompletionProvider.kt

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.completion
+
+import com.intellij.codeInsight.completion.CompletionParameters
+import com.intellij.codeInsight.completion.CompletionResultSet
+import com.intellij.codeInsight.lookup.CharFilter
+import com.intellij.openapi.project.Project
+import com.intellij.util.textCompletion.TextCompletionProvider
+import software.aws.toolkits.core.utils.error
+import software.aws.toolkits.core.utils.getLogger
+import software.aws.toolkits.jetbrains.services.lambda.RuntimeGroup
+import software.aws.toolkits.jetbrains.services.lambda.runtimeGroup
+import java.lang.IllegalStateException
+
+class HandlerCompletionProvider(private val project: Project) : TextCompletionProvider {
+
+    private val logger = getLogger<HandlerCompletionProvider>()
+
+    // Suppress since gradle compiler mismatch return types for property delegate and fail build step
+    private val handlerCompletion: HandlerCompletion by lazy {
```

Comment:
As per the above comment - I'd probably have this just be `nullable` and if there was no handler registered for the given language - then it'll act like it's 'not supported'

Reply:
Updated.